cli/cmd/policy.go

```diff
@@ -196,9 +210,16 @@ func inputPolicy(cmd *cobra.Command) (string, error) {
 	return inputPolicyFromEditor(action)
 }
 
-func inputPolicyFromRepo() (policy string, err error) {
-	err = errors.New("NotImplementedError")
-	return
+func inputPolicyFromLibrary(id string) (string, error) {
+	var (
+		lcl *LaceworkContentLibrary
+		err error
+	)
+
+	if lcl, err = LoadLCL(*cli.LwComponents); err != nil {
```

Comment:
same like for queries, why not:
```suggestion
	if lcl, err = cli.LwComponents.LoadLCL(); err != nil {
```

Reply:
Going to merge this down, but let's continue this discussion...